libraries/wallet/operation_printer.cpp

```diff
@@ -135,13 +139,32 @@ std::string operation_printer::operator()(const asset_create_operation& op) cons
 
 std::string operation_printer::operator()(const htlc_redeem_operation& op) const
 {
```

Comment:
Since the code is not visible in the diff, I'm putting the comment here.

The printer for `htlc_create_operation` needs to return the decrypted memo or "unlock to read memo", see the code for `transfer` (best to reuse it to avoid duplicate code):
https://github.com/bitshares/bitshares-core/blob/f161b3884baabf72978b0cfcaf017cd720146879/libraries/wallet/operation_printer.cpp#L86-L92